superdesk/metadata/item.py

```diff
@@ -646,6 +646,15 @@
     'CropBottom': {'type': 'integer'}
 }
 
+def is_attachment_public(attachment):
```

Comment:
imo this should be in `superdesk.attachments` module

Reply:
agree